scripts/test.ts

```diff
@@ -196,7 +196,7 @@ runner.onComplete((success: boolean) => {
 
 
 glob.sync('packages/**/*.spec.ts')
-  .filter(p => !/\/schematics\/.*\/files\//.test(p))
+  .filter(p => !/\/schematics\/.*\/(other-)?files\//.test(p))
```

Comment:
Non-blocking comment:

Yeah I’ve been thinking about it, and I don’t think this is the right fix (we will have other things than « other-files » as a directory name).

I’m starting to think the right fix would be to move files into a subdirectory of files and other-files into a different subdir under files. But that’s more involved in general, so I’ll approve this. It’s a convention.